css/style.css

```diff
@@ -332,6 +333,389 @@ body {
+}
+
+.stories__header {
+  font-size: 22px;
+  line-height: 21px;
+  text-transform: uppercase;
+  font-weight: 700;
+  color: #000;
+  text-align: center;
+  margin-bottom: 44px;
+}
+
+.stories-card {
+  max-width: 307px;
+}
+
+.stories-card__img {
+  margin-bottom: 29px;
+}
+
+.stories-card__img:hover {
+  opacity: .4;
+  cursor: pointer;
+}
+
+.stories-card__img_1024,
+.stories-card__img_768 {
+  display: none;
+}
+
+.stories-card__header {
+  font-size: 18px;
+  line-height: 17px;
+  font-weight: 700;
+  color: #000;
+  margin-bottom: 8px;
+}
+
+.stories-card__text {
+  font-size: 14px;
+  line-height: 17px;
+  font-weight: 400;
+  color: #000;
+  margin-bottom: 18px;
+}
+
+.stories-card__footer {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.stories-card__link {
+  display: inline-block;
+  width: 125px;
+  position: relative;
+  text-decoration: none;
+  font-size: 12px;
+  line-height: 11px;
+  font-weight: 700;
+  text-transform: uppercase;
+  color: #000;
+}
+
+.stories-card__link::after {
+  display: block;
+  content: '';
+  position: absolute;
+  left: 93px;
+  bottom: 2px;
+  background: url('../icons/arrow-2.png');
+  width: 31px;
+  height: 8px;
+}
+
+.stories-card__link:hover {
+  color: #b5b5b5;
+}
+
+.stories-card__link:active {
+  color: #fff;
+  -webkit-text-stroke: 1px rgba(0, 0, 0, .5);
+  -webkit-text-fill-color: rgba(0, 0, 0, .1);
+}
+
+.stories-card__link:hover::after {
+  opacity: .5;
+}
+
+.stories-card__date {
+  font-size: 12px;
+  line-height: 11px;
+  font-weight: 700;
+  text-transform: uppercase;
+  color: #000;
+  opacity: .3;
+}
+
+.stories-card__date:hover {
+  color: #190000;
+  opacity: .6;
+  cursor: pointer;
+}
+
+.stories__subheader {
+  font-size: 18px;
+  line-height: 17px;
+  font-weight: 700;
+  text-align: right;
+  text-transform: uppercase;
+  color: #000000;
+  opacity: 0.3;
+  margin-bottom: 12px;
+}
+
+.stories__gallery {
+  display: flex;
+  margin-bottom: 80px;
+}
+
+.stories__gallery_1024 {
+  display: none;
+}
+
+.stories__gallery_375 {
+  display: none;
+}
+
+.gallery__item:hover {
+  opacity: .4;
+  cursor: pointer;
+}
+
+/* Футер */
+
+.footer {
+  max-width: 1170px;
+  min-height: 297px;
+  background: #000;
+  position: relative;
+}
+
+.footer__box-wrapper {
+  display: flex;
+  padding: 68px 195px 53px 201px;
+}
+
+.footer__box:first-child {
+  margin-right: 115px;
+}
+
+.footer__box:nth-child(2) {
+  margin-right: 72px;
+}
+
+.footer__box:nth-child(3) {
+  margin-right: 85px;
+}
+
+.footer__subheader {
+  font-size: 12px;
+  line-height: 11px;
+  font-weight: 700;
+  color: #fff;
+  margin-bottom: 10px;
+}
+
+.footer__box:last-child .footer__subheader {
+  margin-bottom: 7px;
+}
+
+.footer__link {
+  text-decoration: none;
+  display: block;
+  font-size: 12px;
+  line-height: 12px;
+  font-weight: 100;
+  color: #fff;
+  margin-bottom: 6px;
+}
+
+.footer__box:last-child .footer__link {
+  margin-bottom: 13px;
+}
+
+.footer__link:hover {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector ".footer__link:hover" to come before selector ".footer__box:last-child .footer__link" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5612618)